src/main/java/com/paneedah/weaponlib/EntityProjectile.java

```diff
@@ -113,9 +116,9 @@ public void setPositionAndDirection(double x, double y, double z, float rotation
 
         this.setLocationAndAngles(x, y + (double) thrower.getEyeHeight(), z, rotationYaw, rotationPitch);
 
-        this.posX -= (double) (CompatibleMathHelper.cos(this.rotationYaw / 180.0F * (float) Math.PI) * 0.16F);
+        this.posX -= (double) (CompatibleMathHelper.cos(this.rotationYaw / 180.0F * (float) Math.PI) * (isAim ? 0f : 0.16F));
         this.posY -= 0.10000000149011612D;
-        this.posZ -= (double) (CompatibleMathHelper.sin(this.rotationYaw / 180.0F * (float) Math.PI) * 0.16F);
+        this.posZ -= (double) (CompatibleMathHelper.sin(this.rotationYaw / 180.0F * (float) Math.PI) * (isAim ? 0f : 0.16F));
```

Comment:
I think there is a little logic error here, with the ternary operator the first option is the true one, so this should be (isAim ? 0.16F : 0)